app/src/main/java/com/example/task/adapters/CompletedAdapter.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.task.adapters
+
+class CompletedAdapter {
+}
```

Comment:
No empty classes. Remove this or something.

Reply:
I need to implement that one in upcoming commits. That's why.